docs/source/using/docker-daemon.adoc

```diff
@@ -6,11 +6,6 @@ include::variables.adoc[]
 :toc-title:
 :toclevels: 1
 
-toc::[]
```

Comment:
@hferentschik Why to remove toc from here? It is creating inconsistency from other pages

Reply:
Because it is not needed. Consistency is one thing, but it must make sense. The topic is too short for a TOC to make sense. As the topic might expand we can re-add it.